bundles/core/src/main/java/com/adobe/cq/commerce/core/components/models/teaser/CifTeaser.java

```diff
@@ -0,0 +1,105 @@
+/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ ~ Copyright 2019 Adobe Systems Incorporated
+ ~
+ ~ Licensed under the Apache License, Version 2.0 (the "License");
+ ~ you may not use this file except in compliance with the License.
+ ~ You may obtain a copy of the License at
+ ~
+ ~     http://www.apache.org/licenses/LICENSE-2.0
+ ~
+ ~ Unless required by applicable law or agreed to in writing, software
+ ~ distributed under the License is distributed on an "AS IS" BASIS,
+ ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ ~ See the License for the specific language governing permissions and
+ ~ limitations under the License.
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/
+package com.adobe.cq.commerce.core.components.models.teaser;
+
+import java.util.List;
+
+import org.osgi.annotation.versioning.ProviderType;
+
+import com.adobe.cq.wcm.core.components.models.ListItem;
+
+@ProviderType
+public interface CifTeaser {
+
+    /**
+     * Name of the resource property that defines whether or not the teaser has Call-to-Action elements
+     *
+     * @since com.adobe.cq.wcm.core.components.models 12.4.0
```

Comment:
These comments are all copy and paste from sites core component.

Reply:
mostly yes, but for `PN_ACTION_CATEGORY_ID` and ` PN_ACTION_PRODUCT_SKU` the verbose is changed to describe the property.

Reply:
CommerceTeaser interface now extends core wcm Teaser interface